src/content/docs/en/recipes/dynamically-importing-images.mdx

```diff
@@ -0,0 +1,132 @@
+    - example-image.jpg
+    - example-image-2.jpg
+    - example-image-3.jpg
+</FileTree>
+
+:::note
+`assets` is a popular folder name convention for placing images but you are free to name the folder whatever you like.
+::: 
+
+2. Create a new Astro component and import the `<Image>` component.
+
+    ```astro title="src/components/MyCustomCardComponent.astro" 
+    ---
+     import { Image } from 'astro:assets';
+    ---
+    ```
+
+3. Create the `props` for your component. 
+
+    ```astro title="src/components/MyCustomCardComponent.astro" ins={4-11}
+    ---
+     import { Image } from 'astro:assets';
+
+     interface Props {
+        imagePath: string;
+        altText: string;
+        name: string;
+        age: number;
+     }
+
+     const { imagePath, altText, name, age } = Astro.props;
+    ---
+    ``` 
+
+4. Create a new `images` variable and use the `import.meta.glob` function which returns an object of all of the image paths inside the `assets` folder. 
+
+    ```astro title="src/components/MyCustomCardComponent.astro" ins={12}
+    ---
+     import { Image } from 'astro:assets';
+
+     interface Props {
+        imagePath: string;
+        altText: string;
+        name: string;
+        age: number;
+     }
+     
+     const { imagePath, altText, name, age } = Astro.props;
+     const images = import.meta.glob<Record<string, () => Promise<{ default: ImageMetadata }>>('/src/**/*.{jpeg,jpg,png,gif}')
+    ---
+    ```
+
+5. Create the markup for your component. 
+
+    ```astro title="src/components/MyCustomCardComponent.astro" ins={14-18}
+    ---
+     import { Image } from 'astro:assets';
+
+     interface Props {
+        imagePath: string;
+        altText: string;
+        name: string;
+        age: number;
+     }
+     
+     const { imagePath, altText, name, age } = Astro.props;
+     const images = import.meta.glob<Record<string, () => Promise<{ default: ImageMetadata }>>('/src/**/*.{jpeg,jpg,png,gif}')
+    ---
+    <div class="card">
+        <h2>{developer}</h2>
+        <p>Age: {age}</p>
+        <Image src={} alt={altText} />
+    </div>
+    ```
+
+6. Inside the `src` attribute, pass in the `images` object and use bracket notation for the image path. Then make sure to invoke the glob function. 
+   
+    ```astro title="src/components/MyCustomCardComponent.astro" ins={17}
+    ---
+     import { Image } from 'astro:assets';
+
+     interface Props {
+        imagePath: string;
+        altText: string;
+        name: string;
+        age: number;
+     }
+     
+     const { imagePath, altText, name, age } = Astro.props;
+     const images = import.meta.glob<Record<string, () => Promise<{ default: ImageMetadata }>>('/src/**/*.{jpeg,jpg,png,gif}')
+    ---
+    <div class="card">
+        <h2>{developer}</h2>
+        <p>Age: {age}</p>
+        <Image src={images[imagePath]()} alt={altText} />
+    </div>
+    ```
+
+7. Import the component inside an Astro page and pass in the values for the `props`. 
```

Comment:
```suggestion
7. Import and use the card component inside an Astro page, passing in the values for the `props`. 
```
I'm a big fan of "import and use" or "import and render" to emphasize that there's going to be two parts to this process.